great_expectations/expectations/expectation.py

```diff
@@ -338,7 +344,7 @@ class Expectation(metaclass=MetaExpectation):
         "result_format",
     )
     default_kwarg_values: ClassVar[
-        dict[str, bool | str | float | RuleBasedProfilerConfig | None]
+        dict[str, Union[bool, str, float, RuleBasedProfilerConfig, None]]
```

Comment:
Does pydantic not allow the other syntax I'm assuming?

Reply:
It relies on the types to validate at runtime.
The minimum version in OSS isn't python 3.8, which doesn't support the alternative union syntax.

Reply:
Thanks!